src/snippet.js

```diff
@@ -0,0 +1,19 @@
+const {
+  fmtStartCodeBlock,
+  markdownCodeTicks
```

Comment:
Leave a trailing comma on objects
```suggestion
  markdownCodeTicks,
```

Reply:
Also if a file exports a class the filename should be cased. Ie: Snippet.js

Reply:
I don't see this happening with any other node modules?

Reply:
OK nvm I found other articles that mention this as well.